portfolio/src/main/webapp/style.css

```diff
@@ -69,11 +69,11 @@ img {
 body {
     display: block;
     margin: 8px;
-    height: 100vw;
+    height: 100vh;
 }
 
 #mail-icon {
-    width: 90%;
+    width: 5vw;
     height: auto;
 }
 
```

Comment:
Separately, maybe worth considering adding something like https://stackoverflow.com/questions/38056679/how-to-change-cursor-on-hover-at-image

so that when people hover over the mail icon, it changes the mouse cursor to be a pointer, so that it's obvious that it's clickable.